src/components/list/OrderedList.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import OrderedList from "./OrderedList";
+import ListItem from './ListItem.tsx';
+
+<Meta
+  title="Components/Data display/Ordered list"
```

Comment:
I would change the placement of all 3 list components

Components/Data display/List/Ordered list
Components/Data display/List/Unordered list
Components/Data display/List/Unstyled list (that is the one that already exists)